core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -135,7 +135,7 @@ $(document).ready(function() {
       }
 
       var data = {"applications": array}
-      $.get("/static/historypage-template.html", function(template) {
+      $.get("static/historypage-template.html", function(template) {
```

Comment:
After discussing with @s-urbaniak directly, I see it's a bug where the link doesn't work when application proxy is set.